libevm/VMOpt.cpp

```diff
@@ -63,14 +63,16 @@ void VM::copyCode(int _extraBytes)
 
 void VM::optimize()
 {
+	copyCode(33);
```

Comment:
In which case does it need much more? The loop continues while ` pc < nBytes` and the only case I see when `pc` ends up > `nBytes` is if `pc` is incremented in one of the lines 09, 107, 112 and the loop ends on that.